New log file opened at Thu Oct 20 19:59:17 PDT 2016 by 24472@Jerry
Taking next connection to: 63050
New client attempting to connect on port: 63051-63050
  Address = /192.168.56.1:63050
This client on port 63051 has uuid 42b3b6b8-dcd7-4668-936b-5ec004186468 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-42b3b6b8-dcd7-4668-936b-5ec004186468
Taking next connection to: 63050
New client attempting to connect on port: 63052-63050
  Address = /192.168.56.1:63050
Handshake exception on socket: 63052
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 63050
New client attempting to connect on port: 63053-63050
  Address = /192.168.56.1:63050
This client on port 63053 has uuid 099e2c6c-415f-484b-9f74-c3ffb8dd11c4 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-42b3b6b8-dcd7-4668-936b-5ec004186468, activator-099e2c6c-415f-484b-9f74-c3ffb8dd11c4
Taking next connection to: 63050
New client attempting to connect on port: 63147-63050
  Address = /192.168.56.1:63050
Handshake exception on socket: 63147
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 63050
New client attempting to connect on port: 63148-63050
  Address = /192.168.56.1:63050
This client on port 63148 has uuid d17f53f2-c817-458c-9b1a-4ea66cc1fdee configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-099e2c6c-415f-484b-9f74-c3ffb8dd11c4, play-fork-run-d17f53f2-c817-458c-9b1a-4ea66cc1fdee
Taking next connection to: 63050
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 63050
